google/cloud/spanner/admin/integration_tests/database_admin_integration_test.cc

```diff
@@ -581,6 +581,69 @@ TEST_F(DatabaseAdminClientTest, DatabasePostgreSQLBasics) {
   EXPECT_STATUS_OK(drop_status);
 }
 
+/// @test Verify dropping a database fails with drop protection enabled.
+TEST_F(DatabaseAdminClientTest, DropDatabaseProtection) {
+  auto instance = database_.instance();
+  auto database =
+      client_
+          .CreateDatabase(
+              instance.FullName(),
+              absl::StrCat("CREATE DATABASE `", database_.database_id(), "`"))
+          .get();
+  ASSERT_THAT(database, IsOk());
+  EXPECT_FALSE(database->enable_drop_protection());
+  EXPECT_FALSE(database->reconciling());
+
+  google::protobuf::FieldMask update_mask;
+  update_mask.add_paths("enable_drop_protection");
+
+  database->set_enable_drop_protection(true);
+  auto updated = client_.UpdateDatabase(*database, update_mask).get();
+  if (emulator_) {
+    EXPECT_THAT(updated, StatusIs(StatusCode::kUnimplemented));
```

Comment:
Consider just skipping the test in this case. We already have tests for `CreateDatabase()` and `DropDatabase()`.

Reply:
The idea there is to get some active notification of when the behavior of the emulator changes.

We do have some instances of `if (emulator) GTEST_SKIP()`, but they are either old, or where the above tactic is too complicated.

Reply:
Okay.